README.md

```diff
@@ -30,16 +30,58 @@ The `PYTHONASYNCIODEBUG` is important to check we have no pending asyncio tasks
 
 Please note that running tests empties the database. DO NOT RUN TESTS in production.
 
-# Generating an OpenAPI document
+# Docker Setup (Recommended)
+
+The easiest way to get started with Folksonomy API is to use Docker. This approach requires minimal setup and provides a consistent development environment.
+
+## Requirements
+- Docker
+- Docker Compose
+
+## Quick Start
+
+1. Clone the repository
+   ```bash
+   git clone https://github.com/openfoodfacts/folksonomy_api.git
+   cd folksonomy_api
+   ```
+
+2. Start the services
+   ```bash
+   docker-compose up -d
+   ```
```

Comment:
Use `docker compose` instead of `docker-compose` (deprecated)

consistently which my comment above, add the migration run (`docker compose run --rm folksonomy_api python db-migration.py`, if we set postgres as a dependency of folksonomy_api)